sefaz/nsu/nsu.go

```diff
@@ -85,19 +85,19 @@ func MustParse(s string) NSU {
 }
 
 // ParseInt parses an integer into an NSU
-func ParseInt(nsu int) (NSU, error) {
-	return Parse(strconv.Itoa(nsu))
+func ParseInt(nsu uint64) (NSU, error) {
```

Comment:
I think it would be better if the name of the function reflected the type, like ParseUint64. The same applies to the other functions

Reply:
done